### ISSUE TITLE
Mission Planner on Android

### ISSUE THREAD

**Issue by Wolfgang-BN** [ArduPilot/MissionPlanner#2211]
Sorry, this isn't an issue,

but Mission Planner is the best of all the ground station controll software!!
Is there any chance, that MP will run on Android in the future? I think, i'm not the only one with this burning desire.

#### Platform
[ X ] All
[  ] AntennaTracker
[  ] Copter
[  ] Plane
[  ] Rover
[  ] Sub

#### Airframe type
All
#### Hardware type
All

**Comment by kikislater** [comment]
C# and .net : no way that it could be on Android ... 
Use Qgroundcontrol

**Comment by meee1** [comment]
not true, ive been working on something in the background. still a while away though

**Comment by kikislater** [comment]
@meee1 it could not be easily port. Glad to hear but will be totally different in term of architecture. Good luck !

**Comment by kikislater** [comment]
So you will use Xamarin ? First project I heard that will use this ... Looking forward how it will be !

**Comment by meee1** [comment]
![image](https://user-images.githubusercontent.com/511568/64109331-ccc6d900-cdb1-11e9-8dad-d6188b27c7e7.png)

**Comment by MR444** [comment]
Happy new year to everyone!

I just found and tested the missionplanner App on Android.
https://play.google.com/store/apps/details?id=com.michaeloborne.MissionPlanner&hl=en
It looks like he put the MP on Android . Not really easy to use without mouse. I tried on 10 inch Lenovo tablet, so I think almost impossible to use on phones.

Connect/Disconnect Button does not work (only once). Crashes sometimes when I want to use missonmode but proably will be improved in future.
Discussions: https://discuss.ardupilot.org/t/mission-planner-android/62896/48
